src/IODD.Parser/Parts/DeviceFunction/StandardVariableRefTParser.cs

```diff
@@ -0,0 +1,43 @@
+using System.Xml.Linq;
+
+using IOLinkNET.IODD.Helpers;
+using IOLinkNET.IODD.Parser;
+using IOLinkNET.IODD.Parser.Parts.Constants;
+using IOLinkNET.IODD.Parts.Constants;
+using IOLinkNET.IODD.Parts.Datatypes;
+using IOLinkNET.IODD.Standard.Structure;
+using IOLinkNET.IODD.Structure.Common;
+using IOLinkNET.IODD.Structure.Datatypes;
+using IOLinkNET.IODD.Structure.DeviceFunction;
+
+namespace IOLinkNET.IODD.Parts.DeviceFunction;
+
+internal class StandardVariableRefTParser
+{
+    private readonly IParserPartLocator _parserLocator;
+
+    public StandardVariableRefTParser(IParserPartLocator parserLocator)
+    {
+        _parserLocator = parserLocator;
+    }
+    public bool CanParse(XName name)
+        => name == IODDDeviceFunctionNames.StandardVariableRefName;
+
+    public VariableT Parse(XElement element)
+    {
+        var variableId = element.ReadMandatoryAttribute("id");
+        var stdVariableCollection = StandardDefinitionReader.GetVariableCollection();
+        var stdVariable = stdVariableCollection?.Elements(IODDStandardDefinitionNames.VariableName).Where(x => x.ReadMandatoryAttribute("id") == variableId).FirstOrDefault();
+
+        DatatypeT? dataType = DatatypeTParser.ParseOptional(stdVariable.Descendants(IODDParserConstants.DatatypeName).FirstOrDefault(), _parserLocator);
```

Comment:
Can you fix the nullability issues here? Maybe it is better to ensure in the StdDefinitionReader that the file is present and contains the desired elements? Otherwise we have to deal with the possible nullability elsewhere.